catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/accesscontrol/AccessControlAccessPlugin.java

```diff
@@ -90,11 +90,18 @@ private static boolean invert(boolean b) {
                   .apply(newMetacard, Core.METACARD_OWNER)
                   .contains(subjectSupplier.get()));
 
-  private final Predicate<Metacard> hasIntrigueTag =
-      metacard ->
-          metacard.getTags().contains(WORKSPACE_TAG)
-              || metacard.getTags().contains(QUERY_TEMPLATE_TAG)
-              || metacard.getTags().contains(ATTRIBUTE_GROUP_TAG);
+  public void setIntrigueTags(List<String> intrigueTags) {
+    this.intrigueTags = intrigueTags;
+  }
+
+  private boolean hasIntrigueTag(Metacard metacard) {
```

Comment:
✏️  you could do something like
```
private final Predicate<Metacard> hasIntrigueTag = 
 metacard ->
    !metacard.getTags().retainAll(intrigueTags).isEmpty()
```

that way you could remove the `this::` you added to the filter below on L161